src/appkit.cs

```diff
@@ -26693,6 +26693,14 @@ interface NSWindowTabGroup
 		void Remove (NSWindow window);
 	}
 
+	[Mac (10,15)]
+	[Native]
+	public enum NSTextScalingType : long
+	{
+		Standard = 0,
+		iOS
```

Comment:
minor: add a `,` after the last member

while it's not required (for the compiler) it will make smaller diff in the future (so it ease reviews)